content/departments/product-engineering/engineering/process/deployments/playbooks.md

```diff
@@ -81,9 +83,13 @@ Usually you'll know the build from which you'd like to deploy, we'll use a speci
 1. Check out the `release` branch
 1. Create a new branch
 1. Run the `update-images.py` script using the image URL from step 3. For example:
+
    ```
    ./update-images.py index.docker.io/sourcegraph/gitserver:118059_2021-11-29_05fcc11@sha256:0c8a862e7977a830e2fa8a690ac243eea1255c150766a44b6c6c86df959d224f
    ```
+
```

Comment:
Wondering if we can/should remove Cloud SaaS and keep just Cloud Devops or maybe Cloud Org?

Reply:
Again here, lets remove Cloud SaaS. 

Quick question - Do you know how much this was actually happening? Were other teams adding our team or devOps to reviews of PRs?

Reply:
Hum, I am not sure how much this was happening, tbh. @unknwon. maybe you know better?